server/appengine/src/main/java/who/PublicWhoServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package who;
+
+// All responses are Client-independent.
+// No Client instance is available in this service.
+public class PublicWhoServiceImpl implements PublicWhoService {
+
+  @Override public GetGlobalStatsResponse getGlobalStats(Void request) {
+    // TODO: Configure cache response headers. RFC 2616 Section 9.5 says we can cache
+    // POST requests.  Or we can enable GET method in present.rpc.RpcFilter.
+
+    // TODO: Get from data source.
+    return new GetGlobalStatsResponse.Builder()
```

Comment:
Present RPC is designed to be protocol independent. Perhaps we could add a ttl field to the response message?

Reply:
done